UnityProject/Assets/Scripts/Items/Stackable.cs

```diff
@@ -293,6 +295,13 @@ public bool WillInteract(InventoryApply interaction, NetworkSide side)
 		//only has logic if this is the target object
 		if (interaction.TargetObject != gameObject) return false;
 
+		//Alt clicking with empty hand calls splitting menu UI
+		if (interaction.IsFromHandSlot && interaction.IsToHandSlot && interaction.FromSlot.IsEmpty && interaction.IsAltClick && side == NetworkSide.Client)
+		{
+			UIManager.Instance.SplittingMenu.Enable();
+			return true;
+		}
```

Comment:
```suggestion
		//Alt clicking with empty hand calls splitting menu UI
		if (side == NetworkSide.Client && interaction.IsFromHandSlot && interaction.IsToHandSlot && interaction.FromSlot.IsEmpty && interaction.IsAltClick)
		{
			UIManager.Instance.SplittingMenu.Enable();
			return true;
		}
```

Lets be nice to the environment and not have the server do checks and waste energy only to fail at the last one :p

Reply:
Okay

Reply:
Also are you sure we dont want to return false here? Or do you want this to be able to trigger on server?

```
else if (CanAccommodate(interaction.UsedObject))
		{
			//combining the stacks
			var sourceStackable = interaction.UsedObject.GetComponent<Stackable>();
			ServerCombine(sourceStackable);
		}
```

Reply:
Returning true cause were moving stack to other hand otherwise. I dont think its terrible solution